bokeh/charts/gmap.py

```diff
@@ -0,0 +1,149 @@
+"""This is the Bokeh charts interface. It gives you a high level API to build
+complex plot is a simple way.
+
+This is the GMap class which lets you build your Google Map charts just
+passing the arguments to the Chart class and calling the proper functions.
+"""
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2014, Continuum Analytics, Inc. All rights reserved.
+#
+# Powered by the Bokeh Development Team.
+#
+# The full license is in the file LICENCE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+from ._chartobject import ChartObject
+from ..models import (Range1d, GMapPlot, GMapOptions)
```

Comment:
No need to use parenthesis if all the imports fit in one line... but it is just style... so no bother if you are not touching this file again...